.github/labeler.yml

```diff
@@ -0,0 +1,50 @@
+# This file is a map of labels to repo glob paths. See https://github.com/marketplace/actions/labeler
+# for more details.
+
+"area: dds":
+  - any: ["/packages/dds/**", "!/packages/dds/sequence/**"]
```

Comment:
i'd rather have both tags. does it support that?

Reply:
Updated.

Reply:
if it doesn't i think i'd prefer the general dds label for now